pkgs/os-specific/linux/systemd/0018-core-don-t-taint-on-unmerged-usr.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: oxalica <oxalicc@pm.me>
+Date: Tue, 4 Oct 2022 09:18:07 +0800
+Subject: [PATCH] core: don't taint on unmerged /usr
+
+NixOS has very different approach towards /bin and /sbin - they don't really exist (except for /bin/sh and /usr/bin/env, because these are used heavily in shebangs around the world).
+The concept of merged or unmerged usr doesn't really apply here at all, it's neither of the two.
+Users don't execute things from /bin or /sbin, there's nothing else in there.
+In all cases, systemd doesn't look things up from /usr/bin or /bin, so showing the taint isn't really helpful.
```

Comment:
Can you reflow that patch to a less large line width?

Reply:
Fixed.